ql/src/test/results/clientpositive/llap/values.q.out

```diff
@@ -10,7 +10,7 @@ POSTHOOK: Input: _dummy_database@_dummy_table
 #### A masked pattern was here ####
 Explain
 CBO PLAN:
-HiveTableFunctionScan(invocation=[inline(ARRAY(ROW(1, 2, 3), ROW(4, 5, 6)))], rowType=[RecordType(INTEGER col1, INTEGER col2, INTEGER col3)])
```

Comment:
Related to the previous comment, then this would not change (it's important it is recognized as ROW).